build.py

```diff
@@ -4,5 +4,21 @@
 if __name__ == "__main__":
     builder = ConanMultiPackager()
     builder.add_common_builds(shared_option_name="gtest:shared", pure_c=False)
+    filtered_builds = []
+    for settings, options in builder.builds:
+        if settings["compiler"] == "Visual Studio":
+            pdbOptions = options.copy()
+            noPdbOptions = options.copy()
+            
+            pdbOptions.update({"gtest:include_pdbs": "True"})
+            noPdbOptions.update({"gtest:include_pdbs": "False"})
```

Comment:
I think this build "noPdbOptions" should not be added, the default is already False, so the "options" build is the same build.